### PR TITLE
fix(curriculum): use variable names not defined automatically by browser in Recipe Ingredient Converter

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

- - -
- Variables corresponding to the HTML elements with id are automatically defined (https://html.spec.whatwg.org/#named-access-on-the-window-object). Because of that unfortunately comparing variable to element does not confirm, that camper defined variable.
- Ie. element `#ingredient` element will be automatically accessible by `ingredient` variable.
- Ids containing `-` don't have such straightforward names to access, since `-` is not valid character for JS variable. (They can be accessed via `window['variable-name-with-dash']`.)
- _Affected_ variables were renamed.

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
Instead of using regex, why don't we change the `id`s to something with a dash in between?

**Comment by gikf** [comment]
> Instead of using regex, why don't we change the `id`s to something with a dash in between?

This or changing the variable name used in lab also would work.

**Comment by ilenia-magoni** [comment]
> or changing the variable name used in lab also would work.

if you can implement this it would be awesoem

**Comment by moT01** [comment]
It looks like this workshop has quite a few steps, so changing the variable name or ID would require a little more of a rework - likely instructional changes, seed and solution code changes, and maybe others. That's probably the better way to go, but I would be fine getting this one in for now as-is - if we don't want to make all those changes.

**Comment by gikf** [comment]
I'll change the names, no worries 🙂

File: curriculum/challenges/english/25-front-end-development/workshop-recipe-ingredient-converter/67353fb237ffbd01af3be1cc.md [hunk 1]
```diff
@@ -7,58 +7,56 @@ dashedName: step-14
 
 # --description--
 
-You can finally start hooking your logic into the user interface. You'll need to query the DOM to get a few elements by ID. Get the elements with the `ingredient`, `quantity`, `unit`, `servings`, `recipe-form`, and `result-list` IDs, and assign each to a variable name that matches the ID.
-
-Remember to use camelCase!
+You can finally start hooking your logic into the user interface. You'll need to query the DOM to get a few elements by ID. Get the elements with the `ingredient`, `quantity`, `unit`, `servings`, `recipe-form`, and `result-list` IDs, and assign them to `ingredientName`, `quantityInUnit`, `unitToConvert`, `numberOfServings`, `recipeForm`, `resultList` variables respectively.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/workshop-recipe-ingredient-converter/67353fb237ffbd01af3be1cc.md:10]:
```suggestion
You can finally start hooking your logic into the user interface. You'll need to query the DOM to get a few elements by ID. Get the elements with the `ingredient`, `quantity`, `unit`, `servings`, `recipe-form`, and `result-list` IDs, and assign them to the variables `ingredientName`, `quantityInUnit`, `unitToConvert`, `numberOfServings`, `recipeForm`, and `resultList`, respectively.
```

**Review by Dario-DC — COMMENTED** [review]
I feel like `quantityInUnit` should be `ingredientQuantity`. Sounds better to me.

**Review by Dario-DC — APPROVED** [review]
LGTM :tada:

**Review by Sembauke — APPROVED** [review]

**Pull request merged**